docs improvement (#8545)
crate/crate: blackbox/docs/appendices/release-notes/unreleased.rst

=======

- Enabled Scalar function evaluation when used :ref:`in the query FROM

>   clause as a table or subquery<table-functions-scalar>`.

Reviewer: ```suggestion
  clause in place of a relation <table-functions-scalar>`.
```
